app/assets/javascripts/tylium/modules/sidebar.js

```diff
@@ -6,50 +6,55 @@
     this.$viewContent = $sidebar.siblings('[data-behavior~=view-content]');
     this.storageKey = $sidebar.data('storage-key');
 
+    // Must match $sidebarCollapsed and $sidebarExpanded values from tyluim/variables.scss
+    this.collapsedWidth = '6rem';
+    this.expandedWidth =  '15.5rem';
+
     this.init();
   }
 
   Sidebar.prototype = {
     init: function() {
-      this.toggle(this.isSidebarOpen(), 'no-animation');
+      this.toggle(this.isSidebarOpen());
 
       var that = this;
-      $('[data-behavior~=sidebar-toggle]').on('click', function() {
-        if (that.isSidebarOpen() && $(this).is('[data-behavior~=open-only]')) return;
 
-        that.toggle(!that.isSidebarOpen(), 'animate');
+      $('[data-behavior~=sidebar-toggle]').on('click', function(e) {
+        if ((that.isSidebarOpen() && $(this).is('[data-behavior~=open-only]')) || ($(e.target).data('behavior') == 'add-node')) return;
```

Comment:
A couple of things on this line:

- Inconsistency between `[data-behavior~=` and `data('behavior')`
- Is `$(e.target)` the same as `$(this)` in this click event?

We can invert this conditional so that we don't have to return:
```
function(e) {
  if (!condition){
    that.toggle(!that.isSidebarOpen());
  }
})...
```

Reply:
I fixed the inconsistency and inverted the conditional. 

`$(e.target)` is a child of `$(this)`, we need to check for it to prevent event bubbling. Otherwise, the collapsed sidebar will expand when a user wants to add a node.